### DISCUSSION TITLE
Proposal to have an animation frame offset for individual engines

### DISCUSSION THREAD

**Discussion by TechCT9902** [endless-sky/endless-sky#8579] [Ideas]
With the current patch, animation frames for engines are perfectly synchronized giving them an uncanny valley appearance. Perhaps a randomized starting frame can be used to help with this?

**Comment by vitalchip** [comment]
"random start frame" is supported but very much like the issue raised for beams weapons it often doesn't look great.

Having a specified offset for each flare would need to be defined by the ship and then the offset passed to the drawlist via engine. Quite a few changes needed there and when you consider a lot of flares are animated using very few frames anyway there will still be synchronized flares on multi engine ships.
